tests/ui/rust-2024/unsafe-extern-blocks/safe-trait.rs

```diff
@@ -0,0 +1,8 @@
+//@ revisions: gated ungated
+#![cfg_attr(gated, feature(unsafe_extern_blocks))]
+
+safe trait Foo {}
+//[gated]~^ ERROR expected one of `!` or `::`, found keyword `trait`
+//[ungated]~^^ ERROR expected one of `!` or `::`, found keyword `trait`
```

Comment:
If these are the same, then they can be combined

```suggestion
//~^ ERROR expected one of `!` or `::`, found keyword `trait`
```